addons/udes_stock/models/stock_picking.py

```diff
@@ -524,18 +524,23 @@ def create_picking(
         confirm=False,
         assign=False,
         create_batch=False,
+        product_quantity=None,
```

Comment:
Need to remove this line, i think have been forgotten when i was testing some issues that were picking all stock quants not the scanned quantities. 
But made a fix when generating products_info so products_info have the needed information.